olm/src/jsMain/kotlin/io/github/matrixkt/olm/Utility.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.matrixkt.olm
+
+
+/**
+ * Olm SDK helper class.
+ */
+public actual class Utility {
+    private val ptr = JsOlm.Utility()
+
+    public actual fun clear() {
+        ptr.free()
+    }
+
+    /**
+     * Verify an ed25519 signature.
+     *
+     * An exception is thrown if the operation fails.
+     * @param key the ed25519 key (fingerprint key)
+     * @param message the signed message
+     * @param signature the base64-encoded message signature to be checked.
+     */
+    public actual fun verifyEd25519Signature(key: String, message: String, signature: String) {
+       ptr.ed25519_verify(key, message, signature)
```

Comment:
Indentation - 7 instead of 8 spaces.

```suggestion
        ptr.ed25519_verify(key, message, signature)
```